packages/core/test/endowmentsToolkit.spec.js

```diff
@@ -457,3 +457,83 @@ test('copyWrappedGlobals - copy from prototype too', (t) => {
 
   t.is(Object.keys(target).sort().join(), 'onTheObj,onTheProto,window')
 })
+
+test('getEndowmentsForConfig - allow redefine on properties', (t) => {
+  'use strict'
+  const knownWritable = new Set(['x', 'y'])
+  const { getEndowmentsForConfig } = prepareTest({ knownWritable })
+  const sourceGlobal = {
+    ...globalThis,
+    x: 1,
+    y: 2,
+    z: 3,
+  }
+  const policy = {
+    globals: {
+      x: 'write+define',
```

Comment:
I'd recommend replacing global policy value strings with constants.  e.g., add `GLOBAL_POLICY_VALUE_WRITE='write'` and `GLOBAL_POLICY_VALUE_REDEFINE='write+define'` to `core/src/constants.js` then consume them everywhere